app/ModelFunctions/AlbumFunctions.php

```diff
@@ -123,61 +133,73 @@ public function create(string $title, int $parent_id, int $user_id): Album
 		return $album;
 	}
 
-	/**
-	 * get the thumbs of an album.
-	 *
-	 * @param array $return
-	 * @param array $album_list
-	 */
-	public function gen_thumbs(array &$return, $album_list)
+	public function get_thumbs_album($album, array $previousThumbsId): BaseCollection
```

Comment:
To match the calling function and to make it clearer that there may be more than one.
```suggestion
	public function get_thumbs_album($album, array $previousThumbIDs): BaseCollection
```